specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2022-10-01-preview/Entities.json

```diff
@@ -1377,6 +1377,18 @@
         "alertType": {
           "type": "string",
           "description": "The name of the alert type."
+        },
+        "intent": {
+          "type": "string",
+          "description": "The intent of the alert."
+        },
+        "techniques": {
```

Comment:
>techniques [](http://example.com/codeflow?start=9&length=10)

Same comment here: can this be changed to an enum of known techniques?

Reply:
This was discussed in earlier stages, unfortunately techniques are not backed by a suitable enum like intent.

Reply:
Are these user provided values that could pretty much be any string?